cli/ops/fs.rs

```diff
@@ -694,7 +695,15 @@ fn op_truncate(
   let is_sync = args.promise_id.is_none();
   blocking_json(is_sync, move || {
     debug!("op_truncate {} {}", path.display(), len);
-    let f = fs::OpenOptions::new().write(true).open(&path)?;
+    let mut open_options = fs::OpenOptions::new();
+    open_options.write(true);
+    if let Some(_mode) = args.mode {
```

Comment:
Don't prefix with underscore, just use `mode`.

Reply:
That's tricky. As implemented, that variable is used only on unix. I had to add the underscore to make the code pass the lint check on windows. You'd think I could just move the `#[cfg(unix)]` to before the `if let...` block, but no, Rust currently doesn't let you put attributes on `if` blocks. What I could do is surround the whole if block in `{ ... }` and then move the `#[cfg(unix)]` to apply to that larger block. Then Windows would never see the `if let(_mode)...` and we could drop the underscore. Do you prefer that? (It seemed 6 of one, half-dozen of the other to me.)